src/test/javascript/spec/component/overview/exercise-details/course-exercise-details.component.spec.ts

```diff
@@ -252,19 +264,17 @@ describe('CourseExerciseDetailsComponent', () => {
         expect(comp.wasSubmissionSimulated).to.be.true;
     });
 
-    it('should simulate a result', () => {
+    it('should simulate a result', fakeAsync(() => {
+        const result = new Result();
+        result.participation = new StudentParticipation();
         comp.exercise = { id: 2 } as Exercise;
         const courseExerciseSubmissionResultSimulationService = fixture.debugElement.injector.get(CourseExerciseSubmissionResultSimulationService);
-        stub(courseExerciseSubmissionResultSimulationService, 'simulateResult').returns(
-            of(
-                new HttpResponse({
-                    body: new Result(),
-                }),
-            ),
-        );
+        stub(courseExerciseSubmissionResultSimulationService, 'simulateResult').returns(of({ body: result } as HttpResponse<Result>));
```

Comment:
Should this be a `jest.stub()`?

Reply:
In the long run, yes, but completely removing sinon will require a lot more refatoring, I would move it to a followup

Reply:
I was just thinking of doing it right now because it was done in other places as well and should be a quick thing to change.

I was not proposing completely removing sinon. What do you think?

Reply:
I adapted it now, however there were nine instances of stubs in the file

Reply:
Thank you.